sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/TagsContainer.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A class representing collection of Tags and their operations.
+    /// </summary>
+    public class TagsContainer : ContainerBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagsContainer"/> class for mocking.
+        /// </summary>
+        protected TagsContainer()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagsContainer"/> class.
+        /// </summary>
+        /// <param name="clientContext"></param>
+        /// <param name="subscriptionId"></param>
+        internal TagsContainer(ClientContext clientContext, string subscriptionId)
+            : base(clientContext, new SubscriptionResourceIdentifier(subscriptionId))
+        {
+            RestClient = new TagsRestOperations(Diagnostics, Pipeline, subscriptionId, BaseUri);
+        }
+
+        /// <summary>
+        /// Gets the valid resource type associated with the container.
+        /// </summary>
+        protected override ResourceType ValidResourceType => SubscriptionOperations.ResourceType;
+
+        /// <summary>
+        /// Gets the operations that can be performed on the container.
+        /// </summary>
+        private TagsRestOperations RestClient;
+
+        /// <summary> This operation allows adding a name to the list of predefined tag names for the given subscription. A tag name can have a maximum of 512 characters and is case-insensitive. Tag names cannot have the following prefixes which are reserved for Azure use: &apos;microsoft&apos;, &apos;azure&apos;, &apos;windows&apos;. </summary>
+        /// <param name="tagName"> The name of the tag to create. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<TagDetails>> CreateOrUpdateAsync(string tagName, CancellationToken cancellationToken = default)
```

Comment:
should return the full [Resource] not the [Resource]Data

Reply:
Done